Client/Frontend/Strings.swift

```diff
@@ -80,7 +80,7 @@ extension Strings {
     public static let SettingsHomePageSectionName = NSLocalizedString("Settings.HomePage.SectionName", value: "Homepage", comment: "Label used as an item in Settings. When touched it will open a dialog to configure the home page and its uses.")
     public static let SettingsHomePageTitle = NSLocalizedString("Settings.HomePage.Title", value: "Homepage Settings", comment: "Title displayed in header of the setting panel.")
     public static let SettingsHomePageUIPositionTitle = NSLocalizedString("Settings.HomePage.UI.Toggle.Title", value: "Show Homepage icon in menu", comment: "Toggle setting to show home page button in menu, or on toolbar.")
```

Comment:
Not directly related to this bug, but I notice the string here doesn't use all caps like all other settings items. Should probably update that.